src/obfl-specification.html

```diff
@@ -1195,6 +1200,72 @@ <h4 id="L722" class="include">evaluate</h4>
           <dd class="markup note">This attribute is only available in the context of a field element.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd class="markup note">For readability, the examples below have plain text
+      instead of braille.</dd>
+    <dd>
+      <p>These are the commonly available variables for use in the <a
+        href="obfl-evaluation-language.html">OBFL Evaluation Language</a>
+        expression:</p>
+      <dl>
+        <dt>page</dt>
+          <dd>The variable holding the current page number. For instance,
+            <code>This is page &lt;evaluate expression="$page"/&gt;</code>.</dd>
+        <dt>volume</dt>
+          <dd>The variable holding the current volume number. For instance,
+            <code>This is volume &lt;evaluate
+            expression="$volume"/&gt;</code>.</dd>
+        <dt>volumes</dt>
+          <dd>The variable holding the total number of volumes. For instance,
+            <code>This book has &lt;evaluate expression="$volumes"/&gt;
+              volumes</code>.</dd>
+        <dt>sheets-in-document</dt>
+          <dd>The variable holding the number of sheets in this document. For
+            instance, <code>This document has &lt;evaluate
+            expression="$sheets-in-document"/&gt; sheets</code>.</dd>
+        <dt>sheets-in-volume</dt>
+          <dd>The variable holding the number of sheets of the current volume.
+            For instance, <code>This volume has &lt;evaluate
+            expression="$sheets-in-volume"/&gt; sheets</code>.</dd>
+      </dl>
+      <p>These common variables can also be used in all use-when conditions.</p>
+    </dd>
+    <dd>
+      <p class="markup">This variable is available in descendant elements of
+        on-volume-start and on-volume-end:</p>
+      <dl>
+        <dt>started-volume-number</dt>
+          <dd>The volume number of the volume that was started or ended.</dd>
+      </dl>
+    </dd>
+    <dd>
+      <p class="markup">This variable is available in descendant elements of
+        on-page-start and on-page-end:</p>
+      <dl>
+        <dt>started-page-number</dt>
+          <dd>The page number of the page that was started or ended.</dd>
+      </dl>
+    </dd>
+    <dd>
+      <p class="markup">This variable is available in descendant elements of
+        toc-entry-on-resumed:</p>
+      <dl>
+        <dt>started-volume-first-content-page</dt>
+          <dd>The first page number of the content of the started volume.</dd>
+      </dl>
+    </dd>
+    <dd>
+      <p class="markup">These variables are available in the cost expression of
+        rendering-scenario:</p>
+      <dl>
+        <dt>min-block-width</dt>
+          <dd>The minimum block with.</dd>
+        <dt>forced-break-count</dt>
+          <dd>The number of forced breaks.</dd>
+        <dt>total-height</dt>
+          <dd>The total height.</dd>
```

Comment:
> Refer to the section "rendering-scenario"

I do that. Please have a look at this temporary available [preview](https://rambags.home.xs4all.nl/obfl/obfl-specification.html#L722) and click on the word "rendering-scenario" just above these lines. It will take you straight to the desired section. See also my previous remark.

> The total height of what?

Please click on "rendering-scenario" in the generated site and it will take you to a more detailed description. This section is meant to be an overview.

Reply:
Again I meant it as in: "Daniel, please refer to ... for more details."

I have to agree with Daniel that these explanations are maybe not so clear, and therefore not very useful. I think the overview is very useful though (I requested it), but maybe just the variable names (the `<dt>`) are enough, as long as there are links to the respective sections.